CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Mapbox welcomes participation and contributions from everyone.
 ## Unreleased
 #### Features
 #### Bug fixes and improvements
+- Added `MapboxNavigation#waypoints` as source of truth for `DirectionsWaypoint`s which can be common for all routes or route specific depending on `RouteOptions#waypointsPerRoute()` parameter. [#6555](https://github.com/mapbox/mapbox-navigation-android/pull/6555)
```

Comment:
```suggestion
- Added `NavigationRoute#waypoints` as the source of truth for `DirectionsWaypoint`s which can be common for all routes or route-specific depending on `RouteOptions#waypointsPerRoute()` parameter. [#6555](https://github.com/mapbox/mapbox-navigation-android/pull/6555)
```

Reply:
The Internet says hypen is not required when the adjective is a predicate (https://english.stackexchange.com/questions/379318/when-is-it-appropriate-to-use-patient-specific-vs-patient-specific).